src/rippled/accountState.js

```diff
@@ -110,13 +111,30 @@ const getAccountState = (account, rippledSocket) => {
         escrows: data[1],
         paychannels: data[2],
         xAddress: decomposedAddress || undefined,
+        deleted: false,
       })),
     )
     .catch((error) => {
       // X-address:
       //   error.toString(): CustomError: account not found
       //   error.code: 404
-      log.error(error.toString())
+      log.error(error)
```

Comment:
Should we be logging this error here if it's expected sometimes? 

(We could log it in an `else` potentially)